Fitness/Views/ProBioPageViewController.swift

```diff
@@ -0,0 +1,301 @@
+            make.leading.trailing.equalToSuperview()
+            make.height.equalTo(2)
+            make.top.equalTo(bioTextView.snp.bottom).offset(36)
+        }
+        
+        expertiseTitleLabel.snp.makeConstraints{ make in
+            make.top.equalTo(bioExpertiseDivider.snp.bottom).offset(23)
+            make.centerX.equalToSuperview()
+        }
+        
+        expertiseTextView.snp.makeConstraints{ make in
+            make.top.equalTo(expertiseTitleLabel.snp.bottom).offset(16)
+            make.leading.equalToSuperview().offset(55)
+            make.trailing.equalToSuperview().offset(-55)
+        }
+        
+        expertiseRoutinesDivider.snp.makeConstraints{ make in
+            make.leading.trailing.equalToSuperview()
+            make.height.equalTo(2)
+            make.top.equalTo(expertiseTextView.snp.bottom).offset(23)
+        }
+        
+        routinesTitleLabel.snp.makeConstraints{ make in
+            make.leading.equalToSuperview().offset(33)
+            make.trailing.equalToSuperview().offset(-33)
+            make.top.equalTo(expertiseRoutinesDivider.snp.bottom).offset(24)
+        }
+        
+        routinesView.snp.makeConstraints{ make in
+            make.leading.equalToSuperview().offset(24)
+            make.trailing.equalToSuperview().offset(-24)
+            make.top.equalTo(routinesTitleLabel.snp.bottom).offset(17)
+        }
+        
+        linksTitleLabel.snp.makeConstraints{ make in
+            make.centerX.equalToSuperview()
+            make.top.equalTo(routinesView.snp.bottom).offset(61)
+        }
+        
+        linksCollectionView.snp.makeConstraints{ make in
+            make.centerX.equalToSuperview()
+            make.top.equalTo(linksTitleLabel.snp.bottom).offset(12)
+            make.height.equalTo(30)
+            make.width.equalTo(30 * pro.links.count + (pro.links.count - 1) * 6)
+            make.bottom.equalToSuperview().offset(-100)
+        }
+    }
+    
+        
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        if collectionView == self.linksCollectionView {
+            return pro.links.count
+        } else {
+            return 0
+        }
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        if collectionView == self.linksCollectionView {
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: PersonalLinkCollectionViewCell.identifier, for: indexPath) as! PersonalLinkCollectionViewCell
+            cell.configure(for: pro.links[indexPath.row])
+            return cell
+        } else {
+            return UICollectionViewCell()
+        }
+    }
+    
+    func numberOfSections(in collectionView: UICollectionView) -> Int {
+        if collectionView == self.linksCollectionView {
+            return 1
+        } else {
+            return 1
```

Comment:
if else return the same value?

Reply:
Returns 1 section for the only collection view that the view controller currently has, and returns 1 as a default. I wrote it that way to account for more view controllers being added in the future but can just set it to return 1 for now.